spec/query_spec.cr

```diff
@@ -91,6 +91,25 @@ describe LuckyRecord::Query do
       count.should eq 1
     end
   end
+
+  describe "#not" do
+    it "negates the given where condition as 'equal'" do
```

Comment:
I think the not works with more than just equal and not equal. So I think this description could be `"negates the given where condition"`

Reply:
Hmm if you are using `name.not("foo")` it is hardcoded to an `EqualCriteria`. Doing an e.g. `ilike` is only possible using the chained version `name.not.ilike("foo")`. Please correct me if I'm wrong!

Reply:
Oh I see. I didn't realize this was just for `not` without changing anything after. I think changing the describe would make it more clear. Maybe `describe "#not with an argument"` Thoughts?